test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -1401,3 +1401,19 @@ test_fail_deny_implicit_indent:
   configs:
     indentation:
       allow_implicit_indents: False
+
+test_pass_templated_newlines:
+  # NOTE: The macro has many newlines in it,
+  # and the calling of it is indented. Check that
+  # this doesn't panic.
+  pass_str: |
```

Comment:
Can you add this test case (or at least confirm it works now)? I noticed this issue yesterday.

```
{% if False %}
SELECT 1
{% else %}
SELECT c
FROM t
WHERE c < 0
{% endif %}
```

In the `main` branch, it only finds and fixes *2* of the 3 bad indentations in the `{% else %}` block:
```
{% if False %}
SELECT 1
{% else %}
SELECT c
    FROM t
    WHERE c < 0
{% endif %}
```

This seems like it may be related to what you've fixed.

Reply:
hmmm - doesn't solve that bug. I'll raise it as a new issue.